### PR TITLE
gitignore file added

### PR DESCRIPTION
**Pull request by murad073**
I have added gitignore file for .net, java, maven, cypress, etc keeping in mind. Whatever ignore-able files are already in master, we will delete them later.

### CODE REVIEW

File: Moshiur-cy/cleanup-db.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+var sql = require("mssql");
+var _ = require('lodash');
+const config = require("./cypress/config/config")
```
Comment (FarzMoho) [inline Moshiur-cy/cleanup-db.js:4]: I did not understand this line...!!!

File: Moshiur-cy/cleanup-db.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+
+var sql = require("mssql");
+var _ = require('lodash');
+const config = require("./cypress/config/config")
+
+//1
+function executeStoredProc() {
```
Comment (gkibria403) [inline Moshiur-cy/cleanup-db.js:7]: Trying to review

File: Moshiur-cy/cleanup-db.js (hunk 1 above)
Comment (murad073) [reply Moshiur-cy/cleanup-db.js:4]: This line actually reads config from a folder location. This is nodejs syntax.

Comment (Ruzinakhatoon) [reply Moshiur-cy/cleanup-db.js:4]: Now I understand.

File: Moshiur-cy/cleanup-db.js (hunk 2 above)
Comment (murad073) [reply Moshiur-cy/cleanup-db.js:7]: Awesome. You are doing great! Keep up the good work.

File: Moshiur-cy/clean-up-dir.bat [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+
+rd /s /q ".\cypress\results\" 
+rd /s /q ".\cypress\screenshots\" 
+rd /s /q ".\cypress\fixtures\" 
+rd /s /q ".\cypress\videos\"
```
Comment (mdzakirh) [inline Moshiur-cy/clean-up-dir.bat:6]: helpful videos

Comment (murad073) [reply Moshiur-cy/clean-up-dir.bat:6]: Yes. These are indeed helpful.

**Pull request merged**